drone_circuit_assets/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(jderobot_assets)
+
+## ROS INSTALL ##
+
+find_package(catkin REQUIRED COMPONENTS)
+
+catkin_package()
+
+install(DIRECTORY models worlds
+ DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+)
+
+## JDEROBOT INSTALL ##
```

Comment:
Optional: jderobot install can be ignored. If we decide to remove it, envvars should be modified to new location